active/PipsPager/PipsPager.md

```diff
@@ -0,0 +1,349 @@
+however can be customized to other glyphs like dashes or squares.
+In the PipsPager control by default, a solid dot will represent each page of the layout. 
+They can be selected by the user to navigate to a specific page.
+
+### Examples
+
+#### Create a PipsPager
+
+A PipsPager with five visible pips that can be selected by the user to jump to that specific page.
+Additionally, the user can use navigational buttons; previous and next, to change the selected page incrementally. 
+Navigating beyond the first or last item does not wrap around.
+By default, the navigational buttons are collapsed, the pips are oriented horizontally,
+and the total number of pages is infinite.
+
+![A prototype of five horizontal dots with the first selected.](images/PipsPager_Default.png)
+
+XAML
+```XAML 
+<muxc:PipsPager x:Name="DefaultPipsPager" />
+```
+
+#### PipsPager with visible navigational buttons
+
+The visibility of the previous and next buttons can be set via the
+`PreviousButtonVisibility` and `NextButtonVisibility` properties. 
+These buttons allow the user to incrementally select a page. The possible values are:
+- Visible: The button is visible and enabled, except it is hidden at the extents.
+  - For example, when the current page is the first page, the previous button is hidden.
+  - Note: Hidden means that the button is invisible but does take up layout space.
+- VisibleOnPointerOver: The behavior is the same as Visible *except* that the button is only Visible when
+the user is hovering over the paging UI with their cursor.
+- Collapsed: The button is not visible to the user and does **not** take up layout space.
+
+![A sketch of five horizontal dots with the first selected and the previous and next buttons are visible](images/PipsPager_Visible.gif)
+
+XAML
+```XAML 
+<muxc:PipsPager x:Name="VisibleButtonPipsPager"
+  NumberOfPages="5"
+  PreviousButtonVisibility="Visible"
+  NextButtonVisibility="Visible" />
+```
+
+#### Vertical PipsPager with VisibleOnPointerOver button visibility
+
+The PipsPager can also be oriented vertically which does not change its behavior or interaction.
+The various button visibility modes can apply to either orientation.
+The below example highlights the VisibleOnPointerOver button visibility.
+
+![A prototype of five vertical dots with the second initially selected and the previous and next buttons appearing and disappearing based on user hover and the selected page](images/PipsPager_VisibleOnPointerOver.gif)
+
+XAML
+```XAML 
+<muxc:PipsPager x:Name="VerticalPipsPager"
+  NumberOfPages="5"
+  Orientation="Vertical" 
+  PreviousButtonVisibility="VisibleOnPointerOver" 
+  NextButtonVisibility="VisibleOnPointerOver" />
+```
+
+#### Number of pages displayed less than total
+
+If the number of pages of content is large and do not need to be navigated to at once,
+the MaxVisiblePips property can be set to limit the number of visible, interactable, pips.
```

Comment:
Checking: the only reason I'd get fewer pips than the max is if there are fewer pages? Or is this like a scroll bar where I might get less if there's less layout space available?

Reply:
@MikeHillberg imagine having 100 pages, we wouldn’t want to display them all (100 pips), so the user can limit that number of displayed pips by setting the Max visible pips number and the Pager will be able to scroll to every consecutive page:)

Reply:
Mike was asking about the case where there are (say) 100 pages, and MaxVisiblePips = 10, but there is only enough room to show 5 pips. Do we show 5 pips? Or do we try to show 10 pips and it's the app's job to provide a scrolling affordance?

Reply:
If there is less space available than MaxVisiblePips (e.g. setting MaxWidth to 50px with MaxVisiblePips of 5 and NumberOfPages > 5) the pips will be cut off and not scroll. I'm not sure if the app would be able to provide a scrolling affordance to navigate to the 4th pip and beyond.
![image](https://user-images.githubusercontent.com/20007011/107427968-d5f3c380-6ad6-11eb-9529-d7adb0624c4f.png)

Reply:
Discussion: If there's not enough layout space for the number of pips it will just clip, not show fewer pips.

Recommend: Call this out in the description.